utils/scapp.h

```diff
@@ -302,20 +302,29 @@ astcEncoderMode(const char* mode) {
                  deterministic).</dd>
       </dl>
     <dt>--normal_mode</dt>
-                 <dd>For ASTC encoder '@b --encode astc' assumes the input texture is
-                 a three component linear LDR normal map storing unit length
-                 normals as (R=X, G=Y, B=Z). The output will be a two component
-                 X+Y normal map stored as (RGB=X, A=Y), optimized for angular
-                 error instead of simple PSNR. The Z component can be recovered
-                 programmatically in shader code by using the equation:
+                 <dd>Only valid for linear textures with two or three components.
+                 If the input texture has three linear components it is assumed to
+                 be a three component linear normal map storing unit length
+                 normals as (R=X, G=Y, B=Z). It will be converted to a two
+                 component X+Y normal map stored as (RGB=X, A=Y) prior to
+                 encoding.  If the input has 2 linear components it is assumed to
+                 be an X+Y normal map.
```

Comment:
s/normal map./map of unit normals./

Reply:
fixed in https://github.com/KhronosGroup/KTX-Software/pull/493/commits/d92aa03a96ee6fa6b858d8e7315e1b0a61f8e5fb